$MirOS: ports/sysutils/ntfsprogs/patches/patch-include_ntfs_types_h,v 1.1.7.1 2005/03/18 15:52:18 tg Exp $
$NetBSD: patch-ab,v 1.1.1.1 2004/10/05 12:39:38 agc Exp $
--- include/ntfs/types.h.orig	Wed Feb 22 14:19:45 2006
+++ include/ntfs/types.h	Sun Apr 29 22:26:06 2007
@@ -59,6 +59,16 @@ typedef u64 sle64;
 typedef u16 ntfschar;			/* 2-byte Unicode character type. */
 #define UCHAR_T_SIZE_BITS 1
 
+/* XXX For some reason, including sys/mount.h on osx defines TRUE and FALSE
+ * so clear them here.
+ */
+#ifdef FALSE
+#undef FALSE
+#endif
+#ifdef TRUE
+#undef TRUE
+#endif
+
 /*
  * Clusters are signed 64-bit values on NTFS volumes.  We define two types, LCN
  * and VCN, to allow for type checking and better code readability.
